driver/csiplugin/controllerserver.go

```diff
@@ -59,8 +60,6 @@ const (
 
```

Comment:
In CreateVolume function, we need to add a check of Scale 5.2.2, as multiple things would be broken with old Scale version. A CG example:
```
if isCGVolume {
		if err := cs.checkCGSupport(assembledScaleversion); err != nil {
			return nil, err
		}
	}
```

Reply:
done